packages/components/lib/utils/platform_utils.dart

```diff
@@ -0,0 +1,34 @@
+import 'dart:io' show Platform;
+import 'package:flutter/foundation.dart' show kIsWeb;
+
+/// PlatforUtils has three methods that returns where is the application is running
+/// isMobile() -> Checks and returns for the mobile(android, IOS)
+/// isDesktop() -> Checks and returns for the desktop(Linux, Windows, MacOS, Fuchsia)
+/// isWeb() -> Checks and returns for the web
```

Comment:
```suggestion
/// isMobile() -> Checks and returns for the mobile(android, IOS)
/// isDesktop() -> Checks and returns for the desktop(Linux, Windows, MacOS, Fuchsia)
/// isWeb() -> Checks and returns for the web
```

these need to be documented on each function